internal/xruntime/runtime_1.22.go

```diff
@@ -0,0 +1,25 @@
+//go:build go1.22
+
+// Copyright (c) 2023 Alexey Mayshev. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package xruntime
+
+import (
```

Comment:
```suggestion
import (
    "rand"
    "time"
)

var (
    r = rand.New(rand.NewSource(time.Now().Unix()))
)
```

Reply:
Why haven't considered using `math/rand/v2`?

Reply:
Yes, it would probably be better this way, then the global variable is not needed.